Views/ResponsiveMediaField.cshtml

```diff
@@ -12,20 +16,21 @@
             <img src="@Model.Media[0].SmallestImageUrl" loading="lazy" alt="@Model.Media[0].MediaText" />
         </picture>
     } else {
-        <ul>
-        @foreach (var media in Model.Media)
-        {
-            <li>
-                <picture>
-                    @foreach (var sourceSet in media.Sources)
-                    {
-                        <source srcset="@sourceSet.Url" media="@($"(min-width: {sourceSet.Breakpoint}px)")" />
-                    }
+        var listClass = string.Format("{0}__media-list--{1}", Model.PartFieldDefinition.PartDefinition.Name, Model.PartFieldDefinition.Name);
```

Comment:
Put this in the top code block, so it's not mixed within the markup and change from `string.Format` to `$` interpolated string.